gatsby/content/blog/2021/12/2021-12-02-cfp-fosdem.mdxxxxx

```diff
@@ -0,0 +1,41 @@
+  categories:
+    - General
+  author: Thib
+  image: https://matrix.org/blog/img/fosem.jpg
+---
+
+## A full day of Matrix talks
+
+This year, the Matrix.org Foundation is excited to host the first ever _Matrix.org Foundation and Community_ devroom at FOSDEM. A full day of talks, demos and workshops around Matrix itself and projects built on top of Matrix.
+
+Matrix is the open source project that publishes the Matrix open standard for secure, decentralised, real-time communication, and its Apache licensed reference implementations.
+
+We encourage people working on the Matrix protocol or building on it in an open source project to submit a proposal! Note that companies are welcome to talk about the Matrix details of their open source projects, but marketing talks are not welcome.
+
+We want this devroom to be a space where the Matrix community can show its work, where developers can talk about the challenges they faced and how they overcame them, and where people can get a glimpse of the future of the Matrix protocol and ecosystem.
+
+## Talk Details
+
+The talks will be pre-recorded in January. They will be played during FOSDEM, followed by a session of live Q&A depending on the format. During the playback of the talk, people will be able to comment and ask questions in the chat (via Matrix!).
+
+The talks can follow one of three formats:
+* 5 min lightning talk, ideal to showcase your project and make people want to have a look at it
+* 20 min talk + 10 min Q&A, for topics that can be covered briefly
+* 50 min talk + 10 min Q&A for more complex subjects which need more focus
+
+We strongly encourage you to prepare a demo when it makes sense, so people can actually see what your work looks like in practice!
+
+Of course, the proposal must respect the FOSDEM terms as well:
+
+> The conference language is English. All content must relate to Free and Open Source Software. By participating in the event you agree to the publication of your recordings, slides and other content provided under the same licence as all FOSDEM content (CC-BY).
+
+## Submitting a Proposal
+
+Proposals must be [submitted on FOSDEM's conference management system Pentabarf](https://penta.fosdem.org/submission/FOSDEM22) before December 17th 2021. If you are not used to Pentabarf, you can follow this [beginners guide to Pentabarf](https://eyskens.me/beginners-guide-to-pentabarf/).
+
+We expect to receive more requests than we have slots available. The devroom organisers (two community members and one core team rep) will be reviewing the proposals and accepting them based on the potential positive impact the project has on Matrix (as defined in by the Mission section of https://matrix.org/foundation).
+
+If a project proposal has been turned down, it doesn't mean we don't believe it has good potential. Maintainers are invited to join the [#twim:matrix.org](https://matrix.to/#/#twim:matrix.org) Matrix room to give it some visibility.
```

Comment:
What is this file?

Reply:
A terrible mistake, I'm going to remove it